src/loader.rs

```diff
@@ -462,31 +477,77 @@ impl<'b, 'w> Loader<'b, 'w> {
 
         trace!("Loading material: {label}");
 
-        let texture = material_obj
-            .transparent_texture()
-            .or_else(|| material_obj.diffuse_texture());
+        let is_transparent = material_obj.transparent_texture().is_some();
+        let transparent_texture = material_obj.transparent_texture();
+        let texture = transparent_texture.or_else(|| material_obj.diffuse_texture());
         let texture = match texture {
             Some(texture) => Some({
-                let texture: Result<_, anyhow::Error> = self.load_texture(texture).await;
+                let ty = "fbx_color_texture";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
                 texture.context("Failed to load diffuse texture")?
             }),
             None => None,
         };
+        let normal_map = match normal_map(&material_obj) {
+            Some(texture) => Some({
+                let ty = "fbx_normal_map";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
+                texture.context("Failed to load normal map")?
+            }),
+            None => None,
+        };
```

Comment:
The code is a bit repetitive here, but I can't use `Option::map` due to the `.await`.